tests/unittests/CoreText/CTLineTests.mm

```diff
@@ -166,24 +166,23 @@
     CFRelease(startTruncated);
 
     CTLineRef endTruncated = CTLineCreateTruncatedLine(line, 40.0, kCTLineTruncationEnd, NULL);
-    EXPECT_EQ(3, CTLineGetGlyphCount(endTruncated));
+    EXPECT_EQ(2, CTLineGetGlyphCount(endTruncated));
 
-    // Truncated should be ABC in one run
+    // Truncated should be AB in one run
```

Comment:
UIFont seems to use Arial as default, so these are the values actually using that font.

---
In reply to: [79909850](https://github.com/Microsoft/WinObjC/pull/1021#discussion_r79909850) [](ancestors = 79909850)